ChangeLog.md

```diff
@@ -9,6 +9,16 @@ SPDX-License-Identifier: MIT
 <!-- Don't forget to update the gas/transaction costs tables in the
 README when a new version is released. -->
 
+* [#155](https://github.com/tqtezos/stablecoin/pull/155)
+  * Added `--description` option to `stablecoin-client`
```

Comment:
Is there an easy way to add a test, that deploys a contract with some description and asserts that the description in the metadata is correct?

Reply:
I tried adding this to `StablecoinClientTest` (and modifying `StablecoinImpl` to pass the `--description ` to `stablecoin-client`), but I just remembered that we can't read a contract's metadata yet, not until morley#346 is completed :/

I've created a new issue so we can do this later (#158) and pushed my work to `diogo/#158-test-metadata`.